types/ember-cli-fastboot/-private.d.ts

```diff
@@ -0,0 +1,5 @@
+export interface FastBoot {
+  require: () => unknown;
```

Comment:
This is mostly right, but needs a module ID to resolve!

```suggestion
  require: (id: string) => unknown;
```